algo/src/main/java/com/hartwig/serve/sources/vicc/ViccExtractor.java

```diff
@@ -314,15 +311,15 @@ private static Set<ActionableHotspot> extractActionableHotspots(@NotNull ViccExt
     }
 
     @NotNull
-    private static <T extends RangeAnnotation> Set<ActionableRange> extractActionableRanges(@NotNull ViccExtractionResult extraction,
-            @NotNull Map<Feature, List<T>> rangesPerFeature) {
+    private static <T extends RangeAnnotation> Set<ActionableRange> extractActionableRange(@NotNull ViccExtractionResult extraction,
+                                                                                            @NotNull Map<Feature, List<T>> rangesPerFeature) {
         Set<ActionableRange> actionableRanges = Sets.newHashSet();
         for (Map.Entry<Feature, List<T>> entry : rangesPerFeature.entrySet()) {
-            List<T> ranges = entry.getValue();
-            if (ranges != null) {
+            List<T> exons = entry.getValue();
+            if (exons != null) {
```

Comment:
Minor, but should this still be `ranges`?

Reply:
correct